sql/catalyst/src/test/scala/org/apache/spark/sql/types/DataTypeSuite.scala

```diff
@@ -478,4 +483,33 @@ class DataTypeSuite extends SparkFunSuite {
 
     assert(result === expected)
   }
+
+  test("SPARK-30004: Allow UserDefinedType to be merged into a standard DateType") {
```

Comment:
If this is one of improvements, you don't need `SPARK-30004: ` in the prefix.

Reply:
Thank you, I wasn't aware of this convention.